gravitee-apim-rest-api/gravitee-apim-rest-api-idp/gravitee-apim-rest-api-idp-core/pom.xml

```diff
@@ -32,6 +32,10 @@
     <artifactId>gravitee-apim-rest-api-idp-core</artifactId>
     <name>Gravitee.io APIM - Management API - Identity Providers - Core</name>
 
+    <properties>
+        <gravitee-secret-api.version>1.0.0-APIM-7497-api-secrets-SNAPSHOT</gravitee-secret-api.version>
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:**  <gravitee-secret-api.version>************************************</gravitee-secret-ap  
[This comment was created by Aqua Pipeline]